### DISCUSSION TITLE
How can I run a project inside the examples folder?

### DISCUSSION THREAD

**Discussion by GP4cK** [TriPSs/nestjs-query#296] [Q&A]
I'm not familiar with nx or nest for monorepos. I tried a few things like this but I'm stuck...
```sh
cd examples
nest start --tsc basic
```

**Comment by TriPSs** [comment]
We updating the complete repo I only used examples for the E2E tests, so i'm not sure they can still be individually ran, everything is still there but I'm also not quite sure how.

(I think the `tsconfig.json` needs to be changed to `tsconfig.app.json` inside the `nest-cli.json`?)

**Reply by GP4cK** [reply]
I see. I'll try to figure it out. It would make it easier to create minimal reproducible examples.
